flexible_event_config.md

```diff
@@ -63,6 +63,11 @@ We will add two optional parameters to the JSON in `Attribution-Reporting-Regist
     // e.g. [5, 10, 100] encodes the following ranges:
     // [[0, 4], [5, 9], [10, 99], [100, MAX_INT]]
     //
+    // At the end of each reporting window, triggers will be summarized into an
+    // integer which slots into one of these ranges. Reports will be sent for
+    // every new range boundary that is crossed. Reports will never be sent for
```

Comment:
What does `new` mean in this context?

Reply:
"new" relative to the ranges you've previously visited. Here is the high level algorithm:
- At the beginning of a window, mark the range you are in $R_b$. For the first window, this will always be the "null" range with 0 in it.
- At the end of a window, mark the range you are in $R_e$
- Emit reports for all bucket ranges in $(R_b, R_e]$

LMK if you can think of a better way of briefly describing this.